_posts/2021-05-25-rust-and-tell.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Rust and Tell"
+date: "2021-05-25 19:00:00 +0200"
+categories: meetup
+location: online
+links:
+    "Meetup.com": https://www.meetup.com/Rust-Berlin/events/xxx
+talks:
+- title: "5 Traits for Beginners to Learn"
+  speaker:
+    name: "Tim McNamara"
+- title: "Ergonomic error handling in Rust"
+  speaker:
+    name: "Senyo Simpson"
+- title: "Building a cryptocurrency derivative exchange with Rust"
+  speaker:
+    name: "Konstantin Wünscher & Blas Rodriguez"
+---
+
+We are happy 🥳 🎉 to announce that the "Rust and Tell" online meetup for May will be on Tuesday, 25th!.
+
+**SPEAKERS WANTED 🙏**
```

Comment:
Might want to call out that we would need speakers for after this one?
Looks like you already got 3 talks for May :)

Reply:
Yes